tools/esdlcmd/esdl2ecl.cpp

```diff
@@ -367,15 +374,19 @@ class Esdl2EclCmd : public EsdlConvertCmd
                 "esdl ecl sourcePath outputPath [options]\n"
                 "\nsourcePath must be absolute path to the ESDL Definition file containing the"
                 "EsdlService definition for the service you want to work with.\n"
-                "outputPath must be the absolute path where the ECL output with be created."
+                "outputPath must be the absolute path where the ECL output with be created.\n"
                 "   Options:\n"
                 "      -x, --expandedxml     Output expanded XML files\n"
                 "      --includes            Process all included files\n"
-                "      --rollup              Roll-up all processed includes to single ecl output file"
+                "      --rollup              Roll-up all processed includes to single ecl output file\n"
+                "      --CDE, --compfilesdir HPCC Component files directory (xslt files)\n"
```

Comment:
Since compfilesdir is also abbreviated, might as well just use CDE? Also since it is an abbreviation should use only one dash (and usually would be lower case).  -cde=<directroy>